pginit-scripts/conseil-schema-create.sql

```diff
@@ -0,0 +1,249 @@
+
```

Comment:
if the `conseil.sql` file is a valid script that can be directly executed instead of this one, we can use _that_ as master copy and docker-postgres will refer directly to it.

I'll check if it works as is.

Thanks for pointing out, good one